addons/payment_alipay/controllers/main.py

```diff
@@ -49,9 +49,17 @@ def _alipay_validate_notification(self, **post):
             return 'success'
         return ""
 
-    @http.route('/payment/alipay/return', type='http', auth="public", methods=['GET', 'POST'])
+    @http.route('/payment/alipay/return', type='http', auth="public", methods=['GET', 'POST'], save_session=False)
     def alipay_return(self, **post):
-        """ Alipay return """
+        """ Alipay return
+        The route is flagged with `save_session=False` to prevent Odoo from assigning a new session
```

Comment:
```suggestion
        """ Alipay return
        
        The route is flagged with `save_session=False` to prevent Odoo from assigning a new session
```

Reply:
I was too quick by copying the old commit message..
It's good now :slightly_smiling_face: